dashboard/test/models/script_test.rb

```diff
@@ -220,22 +217,22 @@ def populate_cache_and_disconnect_db
   test 'allow applab and gamelab levels in hidden scripts' do
     Script.add_script(
       {name: 'test script', hidden: true},
-      [{levels: [{name: 'New App Lab Project'}]}] # From level.yml fixture
+      [{scriptlevels: [{levels: [{name: 'New App Lab Project'}]}]}] # From level.yml fixture
     )
     Script.add_script(
       {name: 'test script', hidden: true},
-      [{levels: [{name: 'New Game Lab Project'}]}] # From level.yml fixture
+      [{scriptlevels: [{levels: [{name: 'New App Lab Project'}]}]}] # From level.yml fixture
```

Comment:
is the name change here and below intentional?

Reply:
Nope, fixed.